template_snippets/cloudconfig/g8s_master.yaml

```diff
@@ -39,8 +39,8 @@
     content: |
       [Unit]
       Description=g8s-master-api-certs
-      Requires=docker.service
-      After=docker.service
+      Wants=calico-node.service
```

Comment:
This doesn't make a lot of sense to me. It means that the certs are started after the calico-node but they don't have to wait. `Wants` wouldn't tear down the certs when calico fails btw if that is your intention. Imo you shouldn't think too much about our chains concept we've used with fleet. This was mainly to have a pod-like concept with fleet.